### PR TITLE
add p_R

### PR DESCRIPTION
**Pull request by edknock**

### CODE REVIEW

File: tests/testthat/test-carehomes-check.R [hunk 1]
```diff
@@ -905,22 +978,3 @@ test_that("Individuals can infect in compartment with non-zero transmission", {
   helper("ICU_transmission", "ICU_W_R_unconf", "gamma_ICU_W_R_step")
   helper("ICU_transmission", "ICU_W_R_conf", "gamma_ICU_W_R_step")
 })
-
-
-test_that("No one is hospitalised, no-one recovers in edge case", {
```
Comment (edknock) [inline tests/testthat/test-carehomes-check.R:910]: Have removed this test because it seems to be testing the same thing as the "No one dies in the community if p_G_D is 0" test

**Review by RaphaelS1 — APPROVED** [review]
Missing updated NEWS otherwise looks good to me

**Review by richfitz — APPROVED** [review]
